lib/apiScenario/gen/testRecordingApiScenarioGenerator.ts

```diff
@@ -168,7 +171,7 @@ export class TestRecordingApiScenarioGenerator {
   ): Promise<RawScenario> {
     logger.info(`\nGenerating ${requestTracking.description}`);
     const testScenario: RawScenario = {
-      scenario: requestTracking.description,
+      scenario: requestTracking.description.replace(/\(/g, "_").replace(/\)/g, "_"),
```

Comment:
```suggestion
      scenario: requestTracking.description.replace(/[\(\)]/g, "_"),
```
And any other special characters? like space,`!@#%^&*`?

Reply:
space may in filename, `!@#%^&*` may not